django/middleware/csrf.py

```diff
@@ -124,29 +125,33 @@ def __init__(self, reason):
 
 
 def _sanitize_token(token):
+    """
```

Comment:
The wording of the `_sanitize_token` function is suboptimal if it doesn't return the token. Maybe change to `_check_token` or so?

Reply:
Thanks for starting to take a look, @apollo13! Yes, this is one of the method renames I was planning on doing as part of a follow-up PR (`_check_token_format()` is what I was thinking, to make clear it's not checking against the cookie). I didn't include those here to keep the diff smaller, but I can do this one now as part of this PR if you prefer. Just let me know.

Reply:
Fine with me either way, just wanted to point it out so it is not lost. Maybe we can rename it shortly before merging? (IE when all other reviews are done -- but in the end a second PR is fine as well)

Reply:
Okay, I'll add an additional commit to the PR with this change. That would be easiest I think.